.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,96 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: dashboard CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-dashboard:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Pipenv
+        run: pipx install pipenv
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          cache: "pipenv"
+          cache-dependency-path: "src/dashboard/Pipfile.lock"
+      - name: Install dependencies
+        run: |
+          cd src/dashboard
+          pipenv install -d .
+          
+  lint-dashboard:
+    needs: build-dashboard
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./src/dashboard
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install pipenv
+        run: pipx install pipenv
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          cache: "pipenv"
+          cache-dependency-path: "src/dashboard/Pipfile.lock"
+      - name: Lint with Black
+        run: pipenv run black apps tests
+      - name: Lint with Ruff
+        run: pipenv run ruff check apps tests
+      - name: Lint with MyPy
+        run: pipenv run mypy apps tests
+        env:
+          DASHBOARD_ALLOWED_HOSTS: localhost
+          DASHBOARD_DB_NAME: qualicharge-dashboard
+          DASHBOARD_DATABASE_URL: psql://qualicharge:pass@postgresql:5432/qualicharge-dashboard
+          DASHBOARD_DEBUG: 1
+          DASHBOARD_SECRET_KEY: the_secret_key
+
+  test-dashboard:
+    needs: build-dashboard
+    runs-on: ubuntu-latest
+    services:
+      postgresql:
+        image: timescale/timescaledb-ha:pg14-ts2.14-oss
```

Comment:
I think a standard postgresql instance would be sufficient for this Django application.

Reply:
✅